tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks.LeakDetection/MarkAndCatalogPackages.cs

```diff
@@ -53,10 +53,17 @@ public class MarkAndCatalogPackages : Task
         [Required]
         public ITaskItem[] PackagesToMark { get; set; }
 
+        /// <summary>
+        /// Use this directory instead of the system temp directory for staging.
+        /// Intended for Linux systems with limited /tmp space, like Azure VMs.
+        /// This directory must exist.
```

Comment:
Why not handle this, to avoid surprises?

Reply:
Good point, added handling.